SalaryService.Application/Queries/CompensationsQuery.cs

```diff
@@ -18,7 +18,27 @@ public async Task<IEnumerable<Compensation>> GetCompensationsAsync()
     {
         return await _context
             .Compensations
+            .Include(x => x.Employee)
             .AsNoTracking()
             .ToListAsync();
     }
+
+    public async Task<IEnumerable<Compensation>> GetPersonalCompensationsAsync(string corporateEmail)
+    {
+        return await _context
+            .Compensations
+            .Include(x => x.Employee)
+            .Where(x => x.Employee.CorporateEmail == corporateEmail)
+            .AsNoTracking()
+            .ToListAsync();
+    }
+
+    public async Task<Compensation> GetCompensationByIdAsync(long id)
+    {
+        return await _context
+            .Compensations
+            .Include(x => x.Employee)
+            .Where(x => x.Id == id)
+            .SingleOrDefaultAsync();
```

Comment:
If a method can return null you should use a word `Find` instead `Get` so the method will have a name `FindCompensationByIdAsync`. Also use the following implementation:
```
public async Task<Compensation?> FindCompensationByIdAsync(long id)
{
    return await _context
        .Compensations
        .Include(x => x.Employee)
        .SingleOrDefaultAsync(x => x.Id == id);
}
```